src/main/webapp/js/user-page-loader.js

```diff
@@ -92,9 +92,8 @@ function fetchProfilePic(){
     if(profilePicUrl == ''){
       profilePicUrl = 'This user has not uploaded any profile picture.';
     }else{
-      profilePicUrl = '<img src=\"' + profilePicUrl + '\" />';
+      profilePicUrl = '<img class=\"rounded-circle square\" src=\"' + profilePicUrl + '\" />';
```

Comment:
I also made a [PR](https://github.com/team-45-su-19/team-45-codeu/pull/86) to your branch so that the display pic is not stretched in case of like a portrait picture + some small fixes to my feed picture and your indentation so I think you could merge mine to yours before merging to master?

Reply:
Yep thanks a lot! I've approved your PR.